README.md

```diff
@@ -125,6 +130,33 @@ The last "unscoped" strategy means there's no file- or
 component-local way of deciding if a style *could* be used elsewhere. You should
 not use sCiSSors on packages / projects with that strategy.
 
+## Using `scissors/css_mirroring_transformer`
+
+Required: Css janus [https://github.com/cegov/wiki/tree/master/maintenance/cssjanus]
+          Note: Two versions available - Google's original cssjanus.py and [https://github.com/cssjanus/cssjanus]. Here the transofrmer uses Google's original cssjanus.py
```

Comment:
Indenting and empty lines matter in markdown. Here, just remove the indent and add a line to separate.

Preview: https://github.com/google/dart-scissors/blob/css-mirroring-monam/README.md#using-scissorscss_mirroring_transformer